src/tensor_ops/mod.rs

```diff
@@ -200,7 +200,7 @@ mod slice;
 mod softmax;
 mod sqrt;
 mod square;
-mod stack;
+pub(crate) mod stack;
```

Comment:
`pub(crate)` makes it public only internal to the crate (e.g. dfdx).

I think you'd either want:
1. `pub mod stack` if we wanted to expose the whole stack module
2. `pub use stack::{TryStack, AddDim};` down below instead of exposing the whole module